lib/Serge/Engine/Plugin/parse_rc.pm

```diff
@@ -86,7 +86,7 @@ sub parse {
         if ($orig_str) {
             my $str = $orig_str;
             $str =~ s/""/"/g;
-            $translated_str = &$callbackref($str, undef, $hint, undef, $lang);
+            $translated_str = &$callbackref($str, undef, $hint, undef, $lang, $hint);
```

Comment:
This is not a proper fix. The last parameter is supposed to be a unique key (if available in the file), whereas you just use hint as a key there (which can be constructed in the parser in multiple ways).

I suggest moving this change out of the importer test PR.